### ISSUE TITLE
give my money back

### ISSUE THREAD

**Issue by gggzq** [getlantern/lantern#6511]
I registered lantern pro like two months ago and now it doesn't work in china, now i want my money back. Ironically, i need to use another vpn app to get this request to you.

**Comment by gggzq** [comment]
i am in Chengdu, China.

**Comment by davidbonachera** [comment]
Update your software to the latest version.

**Comment by gggzq** [comment]
mine is the latest version

**Comment by userfirstone** [comment]
![image](https://user-images.githubusercontent.com/32631032/32422471-6f294112-c2db-11e7-8e64-f85d6aa771c4.png)

**Comment by userfirstone** [comment]
![o89gfzpf 20 08j9969x xx](https://user-images.githubusercontent.com/32631032/32422509-b43b0b00-c2db-11e7-8a61-dfa425e5f816.png)

**Comment by gggzq** [comment]
you are kidding,right? i have already signed up

**Comment by userfirstone** [comment]
u can see the time, i tried it one hour ago. I can use it before two days .

**Comment by gggzq** [comment]
yes, i know that, is there anyway i can get my money back? of course, you can charge me for two months.

**Comment by userfirstone** [comment]
the only thing you can do is waiting or give up, because nobody can help you, i also waiting nearly one month . but now it is ok

**Comment by gggzq** [comment]
…they can't take my money and do nothing. it loooks like i need to issue this problem every week in both english and chinese, i will make sure they respect their customers

**Comment by davidbonachera** [comment]
You're using the wrong channel, go inside your app and use the report issue section instead of posting here (it's not a customer support page)

**Comment by gggzq** [comment]
i did it a lot of times and no one contact with me. actually, i did it again 2 hours ago
